<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Comm_Settings_Screen</class>
 <widget class="QWidget" name="HNS_Comm_Settings_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="LE_CommSet_DNS">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QLineEdit:disabled{
	background-color:#6E6E6E;
	color:#414141;
}</string>
   </property>
   <property name="text">
    <string>0.0.0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_CommSet_Back">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>630</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_CommSet_Cancel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>630</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_CommSet_IPAddr">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QLineEdit:disabled{
	background-color:#6E6E6E;
	color:#414141;
}</string>
   </property>
   <property name="text">
    <string>0.0.0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CommSet_IPMode">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QLabel:disabled{
	background-color:#6E6E6E;
	color:#414141;
}</string>
   </property>
   <property name="text">
    <string>STATIC</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CommSet_Subnet_Title">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>380</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SUBNET MASK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>COMMUNICATIONS SETTINGS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CommSet_IPAddr_Title">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP ADDRESS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CommSet_CommType_Title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COMM TYPE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>160</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PORT NUMBER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_CommSet_Save">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>630</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CommSet_DNS_Title">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DNS SERVER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CommSet_CommMode">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SERIAL AND
ETHERNET</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_CommSet_Subnet">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QLineEdit:disabled{
	background-color:#6E6E6E;
	color:#414141;
}</string>
   </property>
   <property name="text">
    <string>0.0.0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CommSet_Gateway_Title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GATEWAY</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CommSet_IPMode_Title">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP TYPE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_CommSet_PortNum">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>200</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QLineEdit:disabled{
	background-color:#6E6E6E;
	color:#414141;
}</string>
   </property>
   <property name="text">
    <string>5000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_CommSet_Gateway">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>235</width>
     <height>127</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color:white;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QLineEdit:disabled{
	background-color:#6E6E6E;
	color:#414141;
}</string>
   </property>
   <property name="text">
    <string>0.0.0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
